src/components/Page/Page.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import './Page.scss';
+import Sidebar from '../Sidebar/Sidebar';
+import { Accordion, AccordionItem, Link } from 'carbon-components-react';
+
+type PageProps = {
+    activeSmartContract: string
+};
+
+type PageState = { 
+    activeSmartContract: string
+}
+
+class Page extends Component<PageProps, PageState> {
+
+    constructor(props: Readonly<PageProps>) {
+        super(props);
+        this.state = {
+            activeSmartContract: this.props.activeSmartContract
+        }
+    }
+
+    public render() {
+        return (
+            <div className="page-container" id="page-container">
+                <Sidebar/>
+                <div className="page-contents" id="page-contents">
+                    <div className="titles-container">
+                        <Link disabled>{this.state.activeSmartContract}</Link>
+                        <h1>Transactions</h1>
+                    </div>
+                    <div className="contents-container" id="contents-container">
+                        <div className="contents-left" id="contents-left">
+                            <p>Welcome to your transaction web view for <Link disabled>{this.props.activeSmartContract}</Link>; use this to submit and evaluate new and existing transactions.</p>
+                            <p>Unsure of where to start? Follow our introductory Getting Started section opposite.</p>
+                            <br/>
+                            <h5>Smart Contracts</h5>
+                            <p>To switch to the web view for a different smart contract, select the smart contract below</p>
+                            <Link disabled>{this.state.activeSmartContract}</Link>
+                            <br/>
+                            <Link>congaContract@0.0.1</Link>
+                        </div>
+                        <div className="contents-right" id="contents-right">
+                            <h5>Getting Started</h5>
+                            <Accordion>
+                                <AccordionItem title="Create a new transaction"></AccordionItem>
```

Comment:
When changing the theme from dark to light, this text doesn't seem to change colour.